tablecodec/tablecodec.go

```diff
@@ -719,6 +723,19 @@ func reEncodeHandle(handle kv.Handle, unsigned bool) ([][]byte, error) {
 	return [][]byte{intHandleBytes}, err
 }
 
+// reEncodeHandleConsiderNewCollation encodes the handle as a Datum so it can be properly decoded later.
+func reEncodeHandleConsiderNewCollation(handle kv.Handle, columns []rowcodec.ColInfo, restoreData []byte, unsigned bool) ([][]byte, error) {
```

Comment:
```suggestion
func reEncodeHandleConsiderNewCollation(handle kv.Handle, columns []rowcodec.ColInfo, restoreData []byte) ([][]byte, error) {
```

The `unsigned` parameter is unused.